pvfactors/geometry/base.py

```diff
@@ -1092,39 +1100,16 @@ class BasePVArray(object):
     registry_cols = ['geom', 'line_type', 'pvrow_index', 'side',
                      'pvsegment_index', 'shaded', 'surface_index']
 
-    def __init__(self, list_pvrows=[], ground=None, distance=None,
-                 height=None):
-        """Initialize PV array.
+    def __init__(self, axis_azimuth=None):
+        """Initialize Base of PV array.
 
         Parameters
         ----------
-        list_pvrows : list of :py:class:`~pvfactors.geometry.pvrow.PVRow`, optional
-            List of PV rows in the PV array
-            (Default = [])
-        ground : :py:class:`~pvfactors.geometry.pvground.PVGround`, optional
-            Ground geometry for the PV array
-        distance : float, optional
-            Unique distance between PV rows, if exists (Default = None)
-        height : float, optional
-            Unique height of all PV rows, if exists (Default = None)
+        axis_azimuth : float, optional
+            Azimuth angle of rotation axis [deg] (Default = None)
         """
-        self.pvrows = list_pvrows
-        self.ground = ground
-        self.distance = distance
-        self.height = height
-
-        # Initialize shading attributes
-        self.illum_side = None
-        # For view factors
-        self.edge_points = []
-
-        # Property related attributes: will not be built unless called
-        self._all_surfaces = None
-        self._dict_surfaces = None
-        self._surface_registry = None
-        self._view_matrix = None
-        self._obstr_matrix = None
-        self._surfaces_indexed = False
```

Comment:
these were used to cache the values of the class properties. But with the new ``OrderedPVArray``, the pv array attributes can change without notice and it doesn't make sense anymore to cache these values: so they are now recalculated every time the properties are called.
(so we try to call the properties only once, or save their values outside if need to be reused)